commands/lsp_metals_execute.py

```diff
@@ -0,0 +1,4 @@
+from LSP.plugin.execute_command import LspExecuteCommand  # TODO: bring to public API
+
+class LspMetalsExecuteCommand(LspExecuteCommand):
```

Comment:
In most places you don't really need the extra code from `LspExecuteCommand` and could use `LspTextCommand` instead.

Maybe create two separate base classes for each? It's a bit confusing to see the *ExecuteCommand* used everywhere.

Reply:
Good point ! often it is used to have `session_name = "metals"` defined. I Should probably used a different base class